doc/contributor/workflow.md

```diff
@@ -153,7 +155,9 @@ developing, such as loading the core library from disk rather than the JAR.
 
 If you are running a Ruby environment manager like `rvm`, `rbenv`, or `chruby`
 please run `rvm use system`, `rbenv system`, or `chruby system` to clear their
-environment variables, so that the correct gems are picked up.
+environment variables, so that the correct gems are picked up. (Note: `jt build` automatically 
+populates a ruby version along with other managed one. This can be activated, for example with 
+`chruby truffleruby-jvm` when managed by `chruby`.)
```

Comment:
I'm thinking we should probably no longer recommend using `... system` there, and just tell people to use `chruby truffleruby-jvm`. The only disdvantage is core files won't be read from the filesystem, but the advantages are far bigger: gems just work, including using their executable and truffleruby is naturally first in PATH. I'll rephrase this paragraph.

Reply:
Right. How about?

```suggestion
- If you are running a Ruby environment manager like `rvm`, `rbenv`, or `chruby`
please run `rvm use system`, `rbenv system`, or `chruby system` to clear their
environment variables, so that the correct gems are picked up. (Note: `jt build` automatically 
populates a ruby version along with other managed one. This can be activated, for example with 
`chruby truffleruby-jvm` when managed by `chruby`.)
+ If you are running a Ruby environment manager like `rvm`, `rbenv`, or `chruby` it is generally recommended to use `truffleruby-jvm` in order to use the appropriate gems and environment variables (for example `rvm use truffleruby-jvm`, `rbenv truffleruby-jvm`, or `chruby truffleruby-jvm`). It's the `jt build` that provides these ruby versions: `truffleruby-jvm`, `truffleruby-jvm-ce` or `truffleruby-native`.
In cases where the system Ruby version is expected, use `rvm use system`, `rbenv system`, or `chruby system`.
```